web_app_drawer/static/src/less/dropdowns.less

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2016 LasLabs Inc.
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+.dropdown-submenu {
```

Comment:
This is how enterprise does it:

![captura de pantalla de 2016-08-26 09-39-17](https://cloud.githubusercontent.com/assets/973709/17997790/0fe2b836-6b71-11e6-8364-938a2161ad50.png)

Thinking that Odoo has a max of 4 menu levels, this does not seem so bad... They seem to use the `.dropdown-header` class for those little grayed titles. It seems undocumented, but certainly works:

    <nav class="collapse navbar-collapse">
        <ul class="nav navbar-nav">
            <li>
                <a href="#">Navbar Item 1</a>
            </li>
            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown">Navbar Item 2<b class="caret"></b></a>
                <ul class="dropdown-menu">
                    <li class="dropdown-header">header</li>
                    <li><a href="#">Navbar Item2 - Sub Item 1</a></li>
                </ul>
            </li>
            <li>
                <a href="#">Navbar Item 3</a>
            </li>
        </ul>
    </nav>

![captura de pantalla de 2016-08-26 09-52-38](https://cloud.githubusercontent.com/assets/973709/17998086/d981733e-6b72-11e6-8996-73b8ac093636.png)